### PR TITLE
Add sentiment

### PR DESCRIPTION
**Pull request by GuohaoOuyang**
- [x] add sentiment to mention model
- [x] add sentiment library to different platforms
- [x] add faces to Mention.js 
- [x] remove unimplemented platforms and update sidebars
- [x] remove setting security tab
- [x] set handleIndividualCompany taskqueue for setting page "ADD" feature

### CODE REVIEW

File: server/controllers/userController.js [hunk 1]
```diff
@@ -104,7 +104,23 @@ const updateUser = async (req, res) => {
...
+};
+
+// @desc    Add company to user companies
+// @route   PATCH /api/users/addcompany
+// @access  Private
+const addCompany = async (req, res) => {
+  try {
+    const user = await User.findByIdAndUpdate(
+      req.user._id,
+      { $push: req.body },
```
Comment (nalanart) [inline server/controllers/userController.js:121]: I can't find any documentation on this $push. Does it just push to the companies array the new company?

File: server/models/userModel.js [hunk 2]
```diff
@@ -27,11 +27,7 @@ const userSchema = mongoose.Schema(
     platforms: {
```
Comment (nalanart) [inline server/models/userModel.js:28]: I was thinking for the demo, maybe we could set them all to true by default! But we can discuss in standup

**Review by nalanart — APPROVED** [review]

File: server/controllers/userController.js (hunk 1 above)
Comment (GuohaoOuyang) [reply server/controllers/userController.js:121]: Yeah, It just appends one company to the companies array.

File: server/utils/sentimentHandler.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+const sentimentHandler = (context) => {
+  const evaluation = sentiment.analyze(context).comparative;
+  if (evaluation > 0) {
+    return "positive";
+  } else if (evaluation < 0) {
+    return "negative";
+  } else {
+    return "neutral";
+  }
+};
```
Comment (nalanart) [inline server/utils/sentimentHandler.js:13]: Looks good. Out of curiosity, does it just calculate it by itself? Or are there variables we can tweak that would affect the thresholds for certain sentiments?

File: server/controllers/userController.js (hunk 1 above)
Comment (GuohaoOuyang) [reply server/controllers/userController.js:121]: check here: https://docs.mongodb.com/manual/reference/operator/update/push/

File: server/utils/sentimentHandler.js (hunk 3 above)
Comment (GuohaoOuyang) [reply server/utils/sentimentHandler.js:13]: Yes, It's will calculate itself. The comparative has a range of -5 to 5, with 0 neutral. Basically, they have a  list of words with different scores. They will calculate by adding them together and divided by the total number of words. We can overwrite the score of words, or add the score to the word we want.

**Pull request merged**